apps/s_time.c

```diff
@@ -240,16 +224,10 @@ int s_time_main(int argc, char **argv)
                                    www_path);
             if (buf_len <= 0 || SSL_write(scon, buf, buf_len) <= 0)
                 goto end;
-            while ((i = SSL_read(scon, buf, sizeof(buf))) > 0 ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_READ ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_WRITE)
```

Comment:
> 
> 
>     ./openssl s_time -www /test -reuse
>     => (yes SSL_session_reused() = 1 here)
>     725 connections in 0.87s; 833.33 connections/user sec, bytes read 2657125
>     725 connections in 31 real seconds, 3665 bytes read per connection
> 
> This one I cannot explain. TLSv1.2 www with session reuse is much worse than without???

Yes, that was before I added the NODELAY flags.
with NODELAY the same test results in much better numbers:

> still tls1.2 -www -reuse
> 28410 connections in 13.43s; 2115.41 connections/user sec, bytes read 104177885
> 28410 connections in 31 real seconds, 3666 bytes read per connection

that was the only data point that changed due to NODELAY.